sdk/servicebus/Azure.Messaging.ServiceBus/src/Administration/TopicProperties.cs

```diff
@@ -43,6 +43,7 @@ internal TopicProperties(CreateTopicOptions options)
             AuthorizationRules = options.AuthorizationRules;
             Status = options.Status;
             EnableBatchedOperations = options.EnableBatchedOperations;
+            EnablePartitioning = options.EnablePartitioning;
```

Comment:
We'll also need to reflect this in the [Model Factory](https://github.com/Azure/azure-sdk-for-net/blob/d2bcf77a2b1ae1af07a915e751af4c68edd196e1/sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/ServiceBusModelFactory.cs#L151) and [Topic Properties Tests](https://github.com/Azure/azure-sdk-for-net/blob/d2bcf77a2b1ae1af07a915e751af4c68edd196e1/sdk/servicebus/Azure.Messaging.ServiceBus/tests/Administration/TopicPropertiesTests.cs).    

Though in the latter, it appears that there is coverage only for the factory.  @JoshLove-msft: is that intentional or should we also have a test to ensure that the options are correctly implied?

Reply:
Yep, good catch. I will go ahead and update these in a follow up PR. Thanks for the contribution @darrenhull.

Reply:
Great, I’m happy to also follow up with a PR to update the Model Factory and the Topic Properties tests. I will create it on Monday. Just out of interest, when would the next nugget package be published? Just wondering if I should use my fork until it is.

Reply:
Thanks @darrenhull we are on a pretty tight timeline for our next release so I will probably create the PR today. We expect the next nuget package to be published next week.